### ISSUE TITLE
Accessibility:List: Group headers within the list items are not reachable via keyboard.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3475]
Description: Group headers within the list items are not reachable via keyboard.

Expected: Group headers within the list items should be reachable via keyboard

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94647375-eec0c100-030d-11eb-9edb-ef3c1101952b.png)

![image](https://user-images.githubusercontent.com/32538291/94647377-f1bbb180-030d-11eb-9513-a68026e55e2a.png)

**Comment by sKudum** [comment]
PR #4106  addressed this issue.hence closing it.
